tests/foreman/ui/test_ansible.py

```diff
@@ -862,3 +863,64 @@ def test_positive_schedule_recurring_hostgroup_job(self):
 
         :expectedresults: Scheduled Job appears in the Job Invocation list at the appointed time
         """
+
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    @pytest.mark.parametrize('setting_update', ['ansible_verbosity'], indirect=True)
+    def test_positive_ansible_job_with_verbose_stdout(
+        self,
+        target_sat,
+        module_org,
+        module_location,
+        module_ak_with_cv,
+        setting_update,
+        registered_hosts,
+    ):
+        """Verify debug mode output for host
+
+        :id: 06e0a95c-530b-11ef-a28c-36dcd2c0c415
+
+        :steps:
+            1. Select two or more hosts
+            2. Run ansible roles on those hosts
+            3. Check the host execution logs
+
+        :expectedresults: The host execution log should show the debug log for expected host instead of the all host present in the inventory.
```

Comment:
```suggestion
        :expectedresults: Ansible job console stdout should mention the verbose logs for expected host instead of the all host present in the inventory.
```

Reply:
updated.